bigchaindb_driver/offchain.py

```diff
@@ -0,0 +1,229 @@
+        owners_before (:obj:`list` | :obj:`tuple` | :obj:`str`, optional):
+            One or more public keys representing the original owner(s)
+            of the asset being created. Only applies for ``'CREATE'``
+            operations. Defaults to ``None``.
+        owners_after (:obj:`list` | :obj:`tuple` | :obj:`str`, optional):
+            One or more public keys representing the new owner(s) of the
+            asset being created or transferred. Defaults to ``None``.
+        asset (:obj:`dict`, optional): The asset being created or
+            transferred. MUST be supplied for ``'TRANSFER'`` operations.
+            Defaults to ``None``.
+        metadata (:obj:`dict`, optional): Metadata associated with the
+            transaction. Defaults to ``None``.
+        inputs (:obj:`dict` | :obj:`list` | :obj:`tuple`, optional):
+            One or more inputs holding the condition(s) that this
+            transaction intents to fulfill. Each input is expected to
+            be a :obj:`dict`. Only applies to, and MUST be supplied for,
+            ``'TRANSFER'`` operations.
+
+    Returns:
+        dict: The prepared transaction.
+
+    Raises:
+        :class:`~.exceptions.BigchaindbException`: If ``operation`` is
+            not ``'CREATE'`` or ``'TRANSFER'``.
+
+    .. important::
+
+        * For ``operation='CREATE'``, ``owners_before`` MUST be set,
+          meanwhile ``owners_after``, ``asset``, amd ``metadata`` MAY be
+          set. The argument ``inputs`` is ignored.
+
+        * For ``operation='TRANSFER'``, ``owners_after``, ``asset``,
+          and ``inputs`` MUST be set, meanwhile  ``mwtadata`` MAY be set.
+          The argument ``owners_before`` is ignored.
+
+    """
+    operation = _normalize_operation(operation)
+    return _prepare_transaction(
+        operation,
+        owners_before=owners_before,
+        owners_after=owners_after,
+        asset=asset,
+        metadata=metadata,
+        inputs=inputs,
+    )
+
+
+def prepare_create_transaction(*,
+                               owners_before,
+                               owners_after=None,
+                               asset=None,
+                               metadata=None):
+    """
+    Prepares a ``"CREATE"`` transaction payload, ready to be
+    fulfilled.
+
+    Args:
+        owners_before (:obj:`list` | :obj:`tuple` | :obj:`str`): One
+            or more public keys representing the original owner(s) of
+            the asset being created.
+        owners_after (:obj:`list` | :obj:`tuple` | :obj:`str`, optional):
+            One or more public keys representing the new owner(s) of the
+            asset being created. Defaults to ``None``.
+        asset (:obj:`dict`, optional): The asset being created. Defaults
+            to ``None``.
+        metadata (:obj:`dict`, optional): Metadata associated with the
+            transaction. Defaults to ``None``.
+
+    Returns:
+        dict: The prepared ``"CREATE"`` transaction.
+
+    """
```

Comment:
Its worth adding and `.. important::` here saying that if `owners_after` is not specified, it will be set to the same as `owners_before`